.github/PULL_REQUEST_TEMPLATE

```diff
@@ -0,0 +1,39 @@
+**What is the purpose of this pull request? (put "X" next to an item, remove the rest)**
+
+• [ ] Documentation update
+• [ ] Bug fix
+• [ ] New feature
+• [ ] Enhancement to an existing feature
+• [ ] Other, please explain:
```

Comment:
`• [ ] Documentation update` -> `- [ ] Documentation update` ?

Reply:
I changed from `- [ ]` to `• [ ]` in 6bfec64 for the reason in https://github.com/MarkBind/markbind/pull/208#issuecomment-385359556.

Reply:
Got it.